src/Accounts/Accounts/Accounts.csproj

```diff
@@ -37,6 +37,7 @@
   <ItemGroup>
     <None Update="StartupScripts\AzError.ps1" CopyToOutputDirectory="PreserveNewest" />
     <None Update="StartupScripts\InitializeAssemblyResolver.ps1" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="StartupScripts\InitializePSStyle.ps1" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Have we compared the difference between adding a new script vs. combine it with an existing script?

Reply:
By way of comparison, adding a new script will be faster at startup and easier to maintain in the future